lib/Controller/RoomController.php

```diff
@@ -1491,4 +1491,26 @@ public function setMessageExpiration(int $seconds): DataResponse {
 
 		return new DataResponse();
 	}
+
+	#[NoAdminRequired]
+	#[RequireLoggedInModeratorParticipant]
+	public function setCanMentionEveryone(): DataResponse {
+		try {
+			$this->roomService->setCanMentionEveryone($this->room, Room::CAN_MENTION_EVERYONE);
+		} catch (\InvalidArgumentException $e) {
+			return new DataResponse([], Http::STATUS_BAD_REQUEST);
+		}
+		return new DataResponse();
+	}
+
+	#[NoAdminRequired]
+	#[RequireLoggedInModeratorParticipant]
+	public function setCanNotMentionEveryone(): DataResponse {
+		try {
+			$this->roomService->setCanMentionEveryone($this->room, Room::CAN_NOT_MENTION_EVERYONE);
+		} catch (\InvalidArgumentException $e) {
+			return new DataResponse([], Http::STATUS_BAD_REQUEST);
```

Comment:
400 is not documented

Reply:
I saw this but looking the code I think that don't will be possible go to this catch because the second argument isn't an user argument, come from a constant. I only put because if in a moment we change the value of constant, will throw an error. Did you think that is necessary to document this exception at this situation?